sql-ballerina/client.bal

```diff
@@ -22,7 +22,7 @@ public type Client client object {
 
     # Queries the database with the query provided by the user, and returns the result as stream.
```

Comment:
```suggestion
    # Queries the database with the query provided by the user and returns the result as a stream.
```

Reply:
Change Added in https://github.com/ballerina-platform/module-ballerina-sql/pull/228/commits/2656f12c62276e15a256c133465cb60304d0b041